scalyr_agent/__scalyr__.py

```diff
@@ -90,6 +91,55 @@
 __is_frozen__ = hasattr(sys, "frozen")
 
 
+# The constants for INSTALL_TYPE, a variable declared down below.
+PACKAGE_INSTALL = 1  # Indicates source code was installed via a package manager such as RPM or Windows executable.
+TARBALL_INSTALL = 2  # Indicates source code was installed via a tarball created by the build_package.py script.
+DEV_INSTALL = 3  # Indicates source code is running out of the original source tree, usually during dev testing.
+
+
+def read_install_info():
+    """
+    Read the 'install_info' file that has to be located near this file. In opposite return empty dict.
+    """
+    install_info_path = os.path.join(os.path.dirname(__file__), "install_info")
```

Comment:
Should we add .json suffix to the file? I guess it's just a matter of personal personal and doesn't matter much :)

Reply:
Yes, I think adding suffixes should make it more clear and there's no significant reason to keep it without extension.